docs/docs/getting-started/installation/aws.md

```diff
@@ -225,6 +229,56 @@ At this point, the frontend and the API services of `Airy Core` should be access
 airy api endpoint
 ```
 
+### HTTPS using Let's Encrypt
+
+You can customize your installation of `Airy Core` to install a custom Traefik ingress controller which has an enabled `Let's Encrypt` capability. The ingress controller will register and renew the certificates for you automatically.
+
+#### Customize your Airy Core installation
+
+To customize your Airy Core installation, you need to create an initial config file using the following command
+
+```sh
+airy create --provider aws --init-only
+```
+
+Then edit your `airy.yaml` file and add the following configuration
+
+```sh
+kubernetes:
+  host: myairy.myhostname.com
+ingress:
+  https: true
+  letsencryptEmail: "mymail@myhostname.com"
+
+```
+
+The `kubernets.host` value should be set to your desired hostname. Configure the e-mail address you want to use for your Let's Encrypt registration under `ingress.letsencryptEmail`.
+
+After setting these parameters, create your `Airy Core` instance with the following option:
+
+```sh
+airy create --provider aws --provider-config hostUpdate=false
```

Comment:
@steffh if the `--provider-config` is problematic (we use it also to overwrite vpcId and instanceType for AWS), I will try to remove it.